Assets/BossRoom/Scripts/Client/Game/Character/ClientCharacterVisualization.cs

```diff
@@ -339,5 +361,27 @@ public bool IsAnimating
                 return false;
             }
         }
+
+        /// <summary>
+        /// Turns off the normal auto-movement mode, allowing ActionFXes to manually move
+        /// the visualization around. To return to normal mode, call <see cref="StartFollowingNetworkTransform"/>().
+        /// </summary>
+        public void StopFollowingNetworkTransform()
```

Comment:
Is this still needed?

Reply:
(This is new code, I added it for this version of the action.) Since the visualization normally follows the networked transform's position, I wanted an explicit way to cleanly stop it from doing so while being driven by an ActionFX.

In this case it's not _technically_ needed because of the long `ExecTimeSeconds` on this action: after 1.3 seconds, all smooth-movement will have finished, so `ClientCharacterVisualization` won't happen to be trying to move the character at the same time the Action is moving it.

I still like having this here as a way to make the behavior contract explicit, but can remove if desired.

Reply:
YAGNI
This is code lying around that's not used and so not tested. This will rot whenever we change the way we're doing movements in the future.
Yes please remove

Reply:
(Will revise when responding to your other feedback)

Reply:
ok